### PR TITLE
feat: 🔥 Displaying Social Media Favicons On Profile Page

### LINKED ISSUE #3505: 🚀 [Feature] Display favicons for social media links and redesign link box on profile page
> [sushidave](https://github.com/sushidave) **Authored by [sushidave](https://github.com/sushidave)**
_<time datetime="2020-04-25T18:22:22Z" title="Saturday, April 25th 2020, 8:22:22 pm +02:00">Apr 25, 2020</time>_

---

## :rocket: Feature

Display the favicons of social media links on the user profile page and redesign link (smaller icon, text next to icon).

### User Problem

1.) Currently, a standard icon is displayed for all social media links on the user profile page. This makes it hard for users to recognise the social media network instantly, especially for the larger networks. However, the respective favicon for each link is displayed on the social media page in the account settings.
2.) The link icon is too large and takes up too much space of the link box.

### Implementation

1.) If there is a favicon, display it instead of the standard icon.
2.) Smaller icon size (same as avatar icon in the followers box) and put text next to it.

### Design & Layout

hc-favicons-on-profile-page

(left: social media page in the account settings | right: links box on the profile page)

### PR DESCRIPTION
**Pull request by nimitbhargava**
### Issues
- fixes #3505

### CODE REVIEW

**Comment by nimitbhargava** [comment]
Local Evidence - 

URL - /settings/my-social-media
image

URL - /profile/u3/jenny-rostock
image

**Comment by Tirokk** [comment]
@nimitbhargava is this related to issue #3813 ?

**Comment by Mogge** [comment]
Yes it is related to #3813, and does also fix it.

**Review by Mogge — CHANGES_REQUESTED** [review]
Great job so far. Everything is working fine and we have a new component for the social media! Is it true, that the only relevant change was using `img` instead of `user-avatar`? I am looking forward to see the tests.

**Review by Tirokk — COMMENTED** [review]
This component looks really nice @nimitbhargava ! 🚀💫

Thanks a lot that you go for this !!! 🍀🤗

I found one thing which maybe came not in by you, but would be nice to have it a more nicely.
When one edit a social media the input field is really small. Would be cool to have it the width of the field for adding a social media:

Bildschirmfoto 2020-12-15 um 17 43 56

File: webapp/components/SocialMedia/SocialMedia.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+</template>
+
+<script>
+export default {
+  name: 'social-media',
+  props: {
+    userName: {},
+    user: {},
+  },
+  methods: {
```
Comment (Mogge) [inline webapp/components/SocialMedia/SocialMedia.vue:28]: Can this be a computed property?

**Review by Mogge — CHANGES_REQUESTED** [review]
Please see the subbranch for hints for the tests

File: webapp/components/SocialMedia/SocialMedia.vue (hunk 1 above)
Comment (nimitbhargava) [reply webapp/components/SocialMedia/SocialMedia.vue:28]: `500 Internal Error` when using computed

**Review by Mogge — APPROVED** [review]
Congratulations! Your first PR will be merged to master. Thank you very much for your contribution.

**Pull request merged**